chessboard/skoved/chessboard.go

```diff
@@ -0,0 +1,59 @@
+package chessboard
+
+// Declare a type named Rank which stores if a square is occupied by a piece - this will be a slice of bools
+type Rank []bool
```

Comment:
You can declare the size here. It simplifies the rest of the code.

Reply:
done :)